### ISSUE TITLE
list of api endpoints?

### ISSUE THREAD

**Issue by airondumael** [clevr/quest-board#2]
will it help if will plan all the api endpoints that we will be needing before starting with the backend?

I think this will help with the consistency of our endpoints.

**Comment by wildtron** [comment]
best way to do a collated list of endpoints?

**Comment by airondumael** [comment]
How about we create first the list of endpoints we think will be needing and create documentation (apidocs?) of what we want it to look like and code from there?

@clevr/prime , any suggestions?

**Comment by cedricabuso** [comment]
I agree with @airondumael. :)

**Comment by wildtron** [comment]
oke, so what's the best util we use to list api endpoints we need?

**Comment by eshleebien** [comment]
apidocs

**Comment by wildtron** [comment]
i mean, we "list" first before we doc

**Comment by wildtron** [comment]
ie. sheets..

**Comment by eshleebien** [comment]
yes ie. Internet explorer :))

**Comment by jekku** [comment]
:+1:

**Comment by airondumael** [comment]
github wiki?

**Comment by wildtron** [comment]
oke, we use github wiki

**Comment by cedricabuso** [comment]
Github wiki :+1:

**Comment by wildtron** [comment]
see [Endpoints](https://github.com/clevr/quest-board/wiki/Endpoints) if this should suffice.

**Comment by airondumael** [comment]
why not dungeon instead of d?

**Comment by wildtron** [comment]
we make it short and simple and single letters look cool. if we have endpoints with same prefix, we add the next letter of that endpoint eg. dungeon - d, deck - de, something like that

**Comment by cedricabuso** [comment]
I think it would be better if we use the whole word for the endpoints to become more readable and to be easily understood on what that endpoint will do.

**Comment by wildtron** [comment]
no!, the whole "world" is too big! we can't do that! :dancers:

**Comment by airondumael** [comment]
I agree with ced, if the api grows it will be harder to understand what that endpoint does.

**Comment by cedricabuso** [comment]
LOL, *word

**Comment by eshleebien** [comment]
i agree with ced too
